packages/vue/index.js

```diff
@@ -0,0 +1,57 @@
+const web = require('@neutrinojs/web');
+const path = require('path');
+const merge = require('deepmerge');
+const loaderMerge = require('@neutrinojs/loader-merge');
+
+const MODULES = path.join(__dirname, 'node_modules');
+
+module.exports = (neutrino, options) => {
+  neutrino.use(web, options);
+  neutrino.options.extensions = ['vue']; // eslint-disable-line no-param-reassign
+
+  neutrino.config.module
+    .rule('vue')
+      .test(neutrino.regexFromExtensions())
+      .use('vue')
+        .loader(require.resolve('vue-loader'))
+        .options(options);
+
+  neutrino.config.when(neutrino.config.module.rules.has('lint'), () => {
+    neutrino.use(loaderMerge('lint', 'eslint'), {
+      plugins: ['vue'],
+      envs: ['node'],
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true
+        }
+      },
+      rules: {
+        'vue/jsx-uses-vars': 2
+      }
+    });
+  });
+
+  if (neutrino.config.plugins.has('stylelint')) {
+    neutrino.config.plugin('stylelint')
+      .tap(([options, ...args]) => [
+        merge(options, {
+          files: ['**/*.vue'],
+          config: {
+            processors: [require.resolve('stylelint-processor-html')],
+            rules: {
+              // allows empty <style> in vue components
+              'no-empty-source': null
+            }
+          }
+        }),
+        ...args
+      ]);
+  }
+
+  neutrino.config
+    .resolve
+      .modules.add(MODULES).end()
+      .extensions.add('.vue').end()
```

Comment:
We should make this automatic from the `web` preset:

```js
neutrino.config.resolve.extensions.merge(neutrino.options.extensions.map(ext => `.${ext}`);
```

Something like that.